### PR TITLE
assignment w2 JS done

### PR DESCRIPTION
**Pull request by AJasem**

### CODE REVIEW

File: 1-JavaScript/Week2/assignment/ex6-totalCost.js [hunk 1]
```diff
@@ -21,22 +21,30 @@ instead!
...
+  beers: 1.99,
+  chips: 1.99,
+  cake: 1.50,
+  redBull: 1.99,
+  nuts: 0.99
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(cartForParty) {
```
Comment (gabsalvo) [inline 1-JavaScript/Week2/assignment/ex6-totalCost.js:31]: It is correct but it is more of a solution you could use in python.
Can you think of how to solve it using Object and library functions on arrays or objects?

File: 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js [hunk 2]
```diff
@@ -30,8 +30,15 @@ const employeeRecords = [
 ];
 
 // ! Function under test
-function filterPrivateData(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function filterPrivateData(employeeRecords) {
```
Comment (gabsalvo) [inline 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js:33]: Nice

**Review by gabsalvo — COMMENTED** [review]
Overall very good, Keep it up, few imperfections and you could do a more extensive use of function libraries. Keep it up!

File: 1-JavaScript/Week2/assignment/ex1-giveCompliment.js [hunk 3]
```diff
@@ -18,13 +18,18 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/1-Java
    Use `console.log` each time to display the return value of the
    `giveCompliment` function to the console.
 -----------------------------------------------------------------------------*/
-function giveCompliment(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function giveCompliment(name) {
+  const compliments = ['great','awesome','fantastic','wonderful','cool',
+'cheerful','grateful','beautiful','gorgeous','thoughtful'];
+ const compliment = compliments[Math.floor(Math.random()*10)];
```
[the remark this answers is not in the document]
Comment (AJasem) [reply 1-JavaScript/Week2/assignment/ex1-giveCompliment.js:24]: sorry I didn't understand the question

[the remark this answers is not in the document]
Comment (gabsalvo) [reply 1-JavaScript/Week2/assignment/ex1-giveCompliment.js:24]: try use compliments.length instead of 10 in the Math random function

**Pull request merged**